pkg/reconciler/v1alpha1/taskrun/validate_resources_test.go

```diff
@@ -0,0 +1,161 @@
+package taskrun_test
+
+import (
+	"testing"
+
+	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/knative/build-pipeline/pkg/reconciler/v1alpha1/taskrun"
+	"github.com/knative/build-pipeline/pkg/reconciler/v1alpha1/taskrun/resources"
+	tb "github.com/knative/build-pipeline/test/builder"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	validBuildSteps = []corev1.Container{{
```

Comment:
This one is not used and `validBuildStepFn` should most likely be renamed to `validBuildSteps` :upside_down_face: 
This make me think I forgot one like that somewhere :stuck_out_tongue_closed_eyes: :sweat_smile:

Reply:
whoops sorry for the copy pasta XD